rest-model/src/main/java/org/jboss/pnc/rest/restmodel/BuildEnvironmentRest.java

```diff
@@ -121,12 +121,12 @@ public void setSystemImageRepositoryUrl(String systemImageRepositoryUrl) {
         this.systemImageRepositoryUrl = systemImageRepositoryUrl;
     }
 
-    public String getImageId() {
+    public String getSystemImageId() {
         return systemImageId;
     }
 
-    public void setImageId(String imageId) {
-        this.systemImageId = imageId;
+    public void setSystemImageId(String systemImageId) {
+        this.systemImageId = systemImageId;
     }
```

Comment:
Nothing has changed with the api. The change I made corrected the generated swagger docs. Prior to my commit the field name in the docs was "imageId" rather than the correct "systemImageId". For some odd reason changing the setter name and param fixed the problem.